lib/interaction_machine.js

```diff
@@ -670,6 +672,7 @@ var InteractionMachine = Eventable.extend(function(self, api, app) {
                 return self.state.continue_session();
             })
             .then(function(continue_session) {
+                opts.helper_metadata = self.state.helper_metadata;
```

Comment:
Thought about it, but I think having continue session be a function that returns a promise was the wrong way to do things. People are more likely to use other information (possibly including the user's response) to set this, so I think people would rather be setting this data at some point. Not very against it though, fine with us going with that approach if you think its better.